<launch>
    <node pkg="my_robot_control" exec="my_robot_wallfollower_node" name="wall_follower" output="screen">
        <param name="distance_laser" value="0.3" type="double"/>
        <param name="speed_factor" value="1.0" type="double"/>
        <param name="forward_speed" value="0.2" type="double"/>
        <param name="backward_speed" value="-0.2" type="double"/>
        <param name="rotation_speed" value="0.3" type="double"/>
    </node>
</launch>